### ISSUE TITLE
Image of the Soviet single-seat lunar lander

### ISSUE THREAD

**Issue by sflynn850** [raceintospace/raceintospace#859]
### Discussed in https://github.com/raceintospace/raceintospace/discussions/588

<div type='discussions-op-text'>

<sup>Originally posted by **peyre** September 29, 2021</sup>
I wonder if we should consider changing the image of the LK Lander.  The one we inherited from BARIS (what it called the L-3) doesn't look like the real thing and, well, I think it looks kind of lame.   Of course, that was because they really didn't know what it looked like at the time.  I totally get that, but don't see why we couldn't/shouldn't update RIS with newer imagery if we can.  Besides, the real-life LK looks so cool!

Recently I changed the image of the Atlas rocket because the one we inherited from BARIS also looked nothing like a real Atlas (and was even lamer), and that also brought it in line with what the mission movies show.  The mission movies for the LK would still look wrong, but it would look right in R&D/Purchasing.  I don't know how feasible it would ever be to replace the LK movies, given the trouble I've had trying to replace N1, LK-700, and Lapot movies.

![rdfull but 54](https://user-images.githubusercontent.com/3118014/135317899-536a2a4e-cc56-4016-a769-e91c034fcb03.png) ![LK](https://user-images.githubusercontent.com/3118014/135317920-be2620a0-5a8a-47c7-bdee-f1325bd182a6.jpg)
</div>

**Comment by peyre** [comment]
Wait, hang on.  I left it open because the movies still show the LK as looking like the pic on the left.  Though I guess there's another Issue for the movies themselves.

**Comment by sflynn850** [comment]
Whoops! I figured I'd accidentally close a thing or two :)

Reopening this, then, and (for now) pulling it from the 2.0 milestone. Copyright will be a bit of a sticky wicket, for sure — a lot of the existing US video should be public domain, but other systems may be somewhat more impenetrable.

**Comment by peyre** [comment]
Yes, I pulled LK videos from random places on the Internet, and the other three largely from YouTube videos.  It'd be a chore sorting out which videos I used where (though I could do that if there was a point to it) and then trying to get permission to use them from their owners.......might not even be worth trying.

**Comment by sflynn850** [comment]
Sounds like a good candidate for Later :)

**Comment by peyre** [comment]
Yes, it's nothing urgent.  It's a would be nice item, maybe very nice because if you're playing the US you see the goofy LK videos in most games, since the AI uses Fritz's backup program approach.

**Comment by peyre** [comment]
DAMN!  I thought I had it.

I used VLC to break out the movies into individual frames, then I modified each frame one at a time.  Then I used ffmpeg to convert the images into a movie running at 8 frames/second, and then VLC to convert the .mp4s to .ogg files.  But the game refuses to play them - when it hits one, it ends the mission movie altogether and goes straight to the endgame. :(  I was so close!

**Comment by peyre** [comment]
I also tried

`ffmpeg -i sv1l_01.mp4 -codec:v libtheora -qscale:v 3 -codec:a libvorbis -qscale:a 3 -f ogv sv1l_01.ogg`

and it returns

`WARNING /home/leon/raceintospace/src/game/mmfile.cpp:359:yuv_to_overlay	unknown/unsupported theora pixel format`

**Comment by peyre** [comment]
Having a bit more success!  I used theora_png2theora, specifically: 

theora_png2theora -v 10 -f 8 -F 1 -k 24 -o s2tw_01.ogg %05d.png

And it works for s2tw_02 and s2tw_01!  They play.   But the other, sv2l_01, doesn't for some reason.  Puzzling - and frustrating! I'm so close.

**Comment by peyre** [comment]
Aha!  I got it.  I had used the video from the two-seat lander and hadn't renamed it.  Fixed now.  Enjoy the more realistic look of the LK lander!

For future reference, to create videos you edit the individual frames together in their own folder, then run the command above (renaming the output file as appropriate).

```
-v              is picture quality (10 max)
-f 8 -F 1       sets frames/second to 8fps
-k 24           makes "key frame" every 24 frames (once/second)
-o sv2l_01.ogg  is of course the name of the output file
%05d.png        tells it that the file names have 5 digits in the name, and an extension of png
```

**Comment by peyre** [comment]
Looks like the reports that this is all done were slightly premature.  The LK still shows the BARIS L-3 image in the side pictures.  I don't know if we can do anything about these though; these may be hard-coded in the game.

<img width="650" height="424" alt="Image" src="https://github.com/user-attachments/assets/2be40c77-c57f-45b6-9c3b-ba3eb9a8b11c" />

**Comment by peyre** [comment]
Whoops, I closed this one by accident.
